packages/core/src/runtime.ts

```diff
@@ -2,6 +2,8 @@ import { TranslationModule, TranslationMessagesByKey } from '@vocab/types';
 
 import { getParsedICUMessages } from './icu-handler';
 
+export { createTranslationFile } from './translation-file';
```

Comment:
Can we not just import from `@vocab/core/translation-file` instead of re-exporting from both runtimes?

Reply:
We could.
It was a preference for keeping the generated code a little protected and avoid multiple import locations.
It also means should a specific runtime need to do anything a little custom it could.
Happy to change to importing if you think it's worth the trade-off in (minimally) more complicated generated .vocab/index.ts code.